project-6/project_6/main.py

```diff
@@ -0,0 +1,198 @@
+# import libraries
+import os
+from sqlalchemy import (
+    create_engine,
+    Column,
+    String,
+    Integer,
+    DateTime,
+    Float,
+)
+from sqlalchemy.orm import Session, declarative_base
+from sqlalchemy.exc import IntegrityError
+import pandas as pd
+from dotenv import load_dotenv
+import logging
+import datetime
+import warnings
+
+warnings.filterwarnings("ignore")
+
+load_dotenv()
+logging.basicConfig(level=logging.INFO)
+
+Base = declarative_base()
+
+
+# create orm sql classes
+class Artwork(Base):
+    """
+    A class to create an Artwork table using ORM classes
+    """
+
+    __tablename__ = "artwork"
+    __table_args__ = {"extend_existing": True}
+
+    ConstituentID = Column(Integer, primary_key=True, index=True)
+    DisplayName = Column(String, nullable=False)
```

Comment:
Will these columns translate to snake_case? If not, snake_case and lower case, please

Reply:
I have changed the values to snake_case